addons/binding/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -467,7 +466,7 @@ public HttpsURLConnection makeRequest(String verb, String url, @Nullable String
                 if (customHeaders != null) {
                     for (String key : customHeaders.keySet()) {
                         String value = customHeaders.get(key);
-                        if (!StringUtil.isBlank(value)) {
+                        if (value != null && !value.matches("[\\s\\n\\f\\t]+")) {
```

Comment:
It was not my intention to use the jsoup library here.
But I would prefer here 
`!StringUtils.isEmpty(value)`
instead of the regex. The function is used a lot of times in the binding and should be enough.